src/directory/index.jsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import Person from "../directory/singleUser.jsx";
+
+const axios = require("axios");
+
+class directory extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      memberList: []
+    };
+    this.getMembersDate = this.getMembersDate.bind(this);
+    this.getMembersDate();
```

Comment:
@Alihaider168 let's use `componentDidMount` for fetching any initialization data for this component.